.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "LiteFarm Codespace",
+  "image": "mcr.microsoft.com/devcontainers/base:bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "18.16.1",
+      "installYarnUsingApt": false
+    },
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {}
+  },
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "cSpell.language": "pl,en",
+        "cSpell.words": ["LiteFarm"],
+        "cSpell.overrides": [
+          {
+            "filename": "packages/api/src/jobs/locales/en/*.*",
+            "language": "en"
+          },
+          {
+            "filename": "packages/api/src/jobs/locales/pl/*.*",
```

Comment:
This probably comes from your organization's specific setup, but we don't currently support pl locale -- could we remove references to this? Actually, I'm thinking it might be better to remove the extensions from this file altogether, since we don't currently have a unified criteria on extensions to use as a team and it's up to each developer to use the tools they deem necessary

Reply:
Yes and no. We were working on polish translation to LiteFarm and it is almost done: https://github.com/Lukasiewicz-Instytut-Lotnictwa/LiteFarm/tree/feature/GL-4-polish-translation.
Expect a pull request soon. But of course this is not important. It just helps to find spelling errors in translation files when working with Visual Studio Code.